src/pages/documentation/manage-spm-dependencies.md

```diff
@@ -0,0 +1,57 @@
+# Manage dependencies using Swift Package Manager
+
+
+The [Swift Package Manager (SPM)](https://www.swift.org/package-manager/) is a tool for managing the distribution of Swift code. It’s integrated with the Swift build system to automate the process of downloading, compiling, and linking dependencies. AEP Swift SDKs support installation using SPM. 
+
+## Swift Package Collection 
+
+In Swift 5.5, the Swift Package Manager (SPM) adds support for [package collections](https://www.swift.org/blog/package-collections). You can configure package collection in Xcode for easy installation of AEP SDKs. The Swift package collection for the Adobe Experience Platform SDKs is available at the [Adobe Open Source site](https://opensource.adobe.com/aepsdk-core-ios/swift/packages/aep.json).
```

Comment:
Are we using just Platform SDKs now? I find the name confusing.

Reply:
I know that the usage of "AEP" is against branding guidelines, so I'm trying to eliminate usage of that here. Unless the product name itself _is_ that, but then that just raises further questions of how that got past branding since (as far as I know), the usage of AEP is a should never use publicly term (despite everyone within Adobe actually using it :p)